android-components/components/ui/tabcounter/src/main/res/drawable/mozac_ui_tabcounter_badge.xml

```diff
@@ -10,6 +10,6 @@
         </shape>
     </item>
     <item
-        android:drawable="@drawable/mozac_ic_private_mode_12"
+        android:drawable="@drawable/mozac_ic_private_mode_circle_fill_stroke_20"
```

Comment:
I think we meant to remove this drawable and replace it with the above icon. Since this was basically getting us the mask with the circle shape.

Reply:
ahh I see now, I'll fix